ci-operator/config/openshift/operator-framework-catalogd/openshift-operator-framework-catalogd-main.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.14
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: openshift/Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14
+  to: catalogd-controller
```

Comment:
Is this an ok image name?

Reply:
In one of those recent upstream commits, I renamed to just "catalogd" for now. Shall we do that here as well? We can always expand back out if upstream adds more images.

Reply:
Sure, I'll make that change